en/identity-server/next/docs/apis/restapis/organization-discovery-config-mgt.yaml

```diff
@@ -148,20 +197,22 @@ components:
         properties:
           type: array
           items:
-            $ref: '#/components/schemas/Properties'
-    Properties:
-      required:
-        - key
-        - value
-      type: object
-      properties:
-        key:
-          type: string
-          example: emailDomain.enable
-        value:
-          type: string
-          example: true
-
```

Comment:
Why this was removed? better to keep this consistent with api-server yaml

Reply:
Updated with [8d1e3e2](https://github.com/wso2/docs-is/pull/4923/commits/8d1e3e23d53b145b04e82f8fcc6506ff77ae6b54)